### PR TITLE
Add docs for vm-manager

### PR DESCRIPTION
**Pull request by xinxin-Esme**
Signed-off-by: xinxin-Esme <xinxin.wan@intel.com>

### CODE REVIEW

File: docs/fields.md [hunk 1]
```diff
@@ -4,24 +4,138 @@ A [sample vm_manager config file](../sample/guest01.ini).
...
+Specify the kind of display to use in QEMU. The GTK window is used by default if not specified.
+options:
+- options: choose one available display to use in QEMU.
+
+
+### [net]
+
+Optional ethernet model, default is e1000, the whole network emulation will be disabled if it set to "none".
+requirements:
+- model: can not be null.
```
[the remark this answers is not in the document]
Comment (xinxin-Esme) [reply docs/fields.md:104]: > 
Do you mean "model" here could be null?

[the remark this answers is not in the document]
Comment (xinxin-Esme) [reply docs/fields.md:104]: > could you also submit a commit to refine BuildNetCmd(). seems the logic is not correct. all these fields should be optional.

Do you mean "model" here could be null?

File: docs/Documentation.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+5. Download/Install qemu with multi-display support. 
+    ```sh
+    wget https://github.com/intel-sandbox/qemu.celadon/archive/refs/heads/main.zip
+    ```
+
+6. Build qemu.celadon 
+    ```sh
+    unzip qemu.celadon-main.zip
+    ```
+    Download [build-qemu script](https://drive.google.com/file/d/1ZrR_INz6JztAfCZ0QwC54U_fAY0NmSoQ/view?usp=sharing) to build qemu.celadon .
```
[the remark this answers is not in the document]
Comment (xinxin-Esme) [reply docs/Documentation.md:50]: > remove this, need more discussion before publish

remove the documentation file or these lines?

[the remark this answers is not in the document]
Comment (YadongQi) [reply docs/Documentation.md:50]: these lines.

File: docs/fields.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (YadongQi) [reply docs/fields.md:104]: yes. if model is null, make default model to e1000

File: docs/Documentation.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
+# What's New
+* vm-manager support multiple display on Civ
+* Vm-manager monitors the state of the Civ guest to ensure that the app runs when civ is ready.
+
+# Getting Started
```
Comment (YadongQi) [inline docs/Documentation.md:5]: For this part, you can refer https://github.com/projectceladon/vm_manager/wiki/User-Guide

File: docs/Documentation.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+2. Create required directories under $HOME/
+    ```sh
+    mkdir ~/11396
+    mkdir ~/11396/aaf
+    mkdir ~/11396/vtpm0
+    ```
+
+## Run Celadon in a virtual machine
+
+1. Download [caas-releasefiles-userdebug.tar.gz](https://mcg-depot.intel.com/artifactory/cactus-absp-jf/celadon-engineering/11396/caas/userdebug/caas-releasefiles-userdebug.tar.gz) in ``$HOME/11396/`` .
```
Comment (YadongQi) [inline docs/Documentation.md:20]: Do not publish internal link. Use https://github.com/projectceladon/celadon-binary/ instead.

**Review by YadongQi — APPROVED** [review]

**Pull request merged**